README.md

```diff
@@ -321,24 +314,36 @@ Both receive the message payload and size. Return values are ProcessMessageResul
 
 ## Source
 
-## StreamMaster
-The StreamMaster receives the array of DemuxTopic from FileWriterCommand and
-instantiates the Streamer array according to the topics. Eventually retrieves
-the list of brokers from Kafka.
+## StreamMaster 
+
+The StreamMaster receives the array of DemuxTopic from
+FileWriterCommand and instantiates the Streamer array according to the
+topics. Eventually retrieves the list of brokers from Kafka.
+
+* `start_time` and `stop_time` can be used to set a the timestamp of
+the fist and last event to be written (caution: actually events
+with timestamp earlier than request will be written);
+* upon a `stop` message the ``Master`` can stop the writing;
+* if a `status-uri` is configured sends a (JSON formatted) status
+  report on the corresponding topic;
+* a global `status` flag report the status of
+``StreamMaster``. Definitions are in
+`Status::StreamMasterErrorCode` (the function `Err2Str`
+converts the error code into a human readable string). Possible values:
+
+| Code                 |
+|----------------------|
+|  not_started         | 
+|  running             |
+|  has\_finished       |
+|  streamer\_error     |
+|  statistics\_failure |
+|  streammaster\_error |
```

Comment:
Would it make sense to put internal technical documentation into (doxygen-enabled) comments into the source files, and leave README for information relevant for the users of the application?

Reply:
:white_check_mark: